README.md

```diff
@@ -56,6 +56,13 @@ package-wheezy/. Build in there (`dpkg-buildpackage -b`)
 
 The wheezy build does not include bladeRF, HackRF, or LimeSDR support.
 
+## Building on FreeBSD
+
+To build, you will need librtlsdr, pkgconf and GNU Make. For users of pkg(8):
+`pkg install rtl-sdr pkgconf gmake`.
```

Comment:
Should this line be?
``` suggestion
`pkg install rtl-sdr bladerf hackrf pkgconf gmake`.
```

Reply:
My understanding is that we'll use the README.md section that's already been committed to the 'dev' branch in e00d12f. At least that's the version I was asking @mutability about yesterday.

Reply:
Yep, the changes in this PR were merged as part of #38